templates/taxi/car_confirm_delete.html

```diff
@@ -7,4 +7,4 @@ <h1>Delete car?</h1>
 
     <input type="submit" value="Yes" class="btn btn-danger">
   </form>
```

Comment:
According to the checklist, there should be a 'Cancel' button in addition to the 'Delete' button. This 'Cancel' button should lead the user back to the previous page they were on. Please add this functionality to improve the user experience.